main/modes/games/cosCrunch/cosCrunch.c

```diff
@@ -62,7 +73,7 @@ swadgeMode_t cosCrunchMode = {
     .fnExitMode               = cosCrunchExitMode,
     .fnMainLoop               = cosCrunchMainLoop,
     .fnAudioCallback          = NULL,
-    .fnBackgroundDrawCallback = cosCrunchBackgroundDrawCallback,
+    .fnBackgroundDrawCallback = NULL,
```

Comment:
You might want this back one day. It's not that hard to `memcpy()` chunks of a background image directly to the framebuffer here.